benchmark/README.md

```diff
@@ -1,41 +1,99 @@
-Swift Benchmark Suite
-=====================
 
-This directory contains the Swift Benchmark Suite.
+# Swift Benchmark Suite
 
-Running Swift Benchmarks
-------------------------
+This directory contains the Swift Benchmark Suite. This is meant to be a suite
+of regression, micro-benchmark, small, and medium sized benchmarks that is used
+to validate the performance of the swift compiler and standard library.
 
-To run Swift benchmarks, pass the `--benchmark` flag to `build-script`. The
-current benchmark results will be compared to the previous run's results if
-available. Results for each benchmark run are logged for future comparison.
+## Building Swift Benchmarks
 
-For branch based development, take a baseline benchmark on the Swift `master`
-branch, switch to a development branch containing potentially performance
-impacting changes, and run the benchmarks again. Upon benchmark completion, the
-benchmark results for the development branch will be compared to the most
-recent benchmark results for `master`.
+There are 3 ways to build the benchmark suite today:
+
+1. In tree using build-script and cmake.
+2. Standalone using swiftpm.
+3. Standalone using cmake.
+
+**NOTE:** There are various level of support here. Specifically:
+
+1. Linux is only supported by the standalone cmake build.
+2. Non macOS Darwin platforms do not today support the swiftpm build.
+3. macOS supports all 3 ways of building.
+
+This is not due to to intention, but rather than more engineering time would be
```

Comment:
"This is not due to intention, but rather, more engineering time would be necessary to produce these builds."

I am not entirely sure what is trying to be said here.

Reply:
"productization" maybe is something that we need to add to the lexicon. Basically the idea is that there are certain things that are "possible" to do with this software today that may either be fragile, unsupported, or not tested. Productization is changing the software so that it is easily usable by others.